src/llvm-pass-helpers.cpp

```diff
@@ -303,7 +309,7 @@ namespace jl_well_known {
                     T_prjlvalue,
                     { Type::getInt8PtrTy(ctx),
                         T_size,
-                        T_size }, // type
+                        T_prjlvalue }, // type
```

Comment:
This change may demand that we allocate GC root, which is generally not especially desirable. Is it actually necessary to change this? These are also not always legal pointers, so it is not generally correct to mark them as such.

Reply:
This was mostly an experiment, I'll roll it back